mantis-connectors/mantis-connector-job/src/main/java/io/mantisrx/connector/job/source/JobSource.java

```diff
@@ -129,6 +133,15 @@ public JobSource(String targetInfoStr) {
         return sourceObs;
     }
 
+    @Override
+    public void close() throws IOException {
+        try {
+            Closeables.combine(jobs).close();
```

Comment:
if this is how we intend to use it, I think it might be better to write this as a utility function. 
Alternatively, we just make `jobs` a `Closeables` and keep on adding to it instead of the array.

Reply:
Isn't Closeables::combine (https://github.com/Netflix/mantis/blob/master/mantis-common/src/main/java/com/mantisrx/common/utils/Closeables.java) a good enough utility function? What more do you want to move into a function?

Regarding the second comment, I prefer to keep it stored in the original DS because if you capture it as a closeable, you lose all other capabilities.